hack/build_speed.sh

```diff
@@ -0,0 +1,81 @@
+#! /bin/sh
+
+# The main goal of this script is test and time builds using Buildah or Docker.
+# We hope to use it to help optimize Buildah build performance
+#
+# It takes two options
+# First option tells the type of the container image
+# build to do.  Valid options are:
+#       Docker  - docker build
+#       Buildah - buildah bud
+#       Both    - Do docker build followed by buildah bud
+#
+# Second Option specifies a directory or cleanup
+# The script will 'find' files beginning with Dockerfile, for each Dockerfile
+# it finds it will run a build with the Dockerfile and directory for the
+# context. When it does the builds, it will call time on them to show how
+# long the builds take.  The created image name will be a combination of the
+# lowercased Directory name that the Dockerfile was found in plus the lower
+# cased dockerfile name.
+#
+# if the second field is cleanup, the script will remove all images from the
+# specified builder.
+#
+# The script does not check for conflicts on nameing.
+#
+
+Dockerfiles() {
+    find -L $1 -name Dockerfile\*
+}
+
+Buildah() {
+    Name=$1
+    Dockerfile=$2
+    Context=$3
+    echo buildah bud --layers --no-cache -f ${Dockerfile} -t ${Name} ${Context};
+    time buildah bud --layers --no-cache -f ${Dockerfile} -t ${Name} ${Context};
+}
+
+Docker() {
+    Name=$1
+    Dockerfile=$2
+    Context=$3
+    echo docker build --no-cache -f ${Dockerfile} -t ${Name} ${Context};
+    time docker build --no-cache -f ${Dockerfile} -t ${Name} ${Context};
+}
+
+Both() {
+    Docker $1 $2 $3
+    Buildah $1 $2 $3
+}
+
+Docker_cleanup() {
+    docker rmi --force $(docker images  -q) 
+}
+
+Buildah_cleanup() {
+    buildah rmi --force --all
+}
+Both_cleanup() {
+    Docker_cleanup
+    Buildah_cleanup
+}
+
+Builder() {
+    Cmd=$1
+    Path=$2
+    for i in $(Dockerfiles ${Path});do
+	echo "DAN"
+	name=$(basename $(dirname $i) | sed -e 's/\(.*\)/\L\1/')
+	name=${name}_$(basename $i | sed -e 's/\(.*\)/\L\1/')
+	echo ${Cmd} $(basename $i) ${name} $(dirname $i)
```

Comment:
@edsantiago beat me to the other one, but this also doesn't work. 

This should:
```
echo ${Cmd} ${name} $i $(dirname $i)
${Cmd} ${name} $i $(dirname $i)
```